<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CREA TU COCHE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Crear">
     <attribute name="title">
      <string>Crear Vehiculo</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Numero de Ruedas:</string>
      </property>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Maticula:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushGenerar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar Matricula</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Portavasos gigante</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushGuardar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>460</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Burbuja para ni√±os</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Kit de reparacion de pinchazos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_Et">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>90</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>120</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Combustible:</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Diesel</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Gasolina</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>98</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Electrico</string>
       </property>
      </widget>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>421</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>90</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiples bocinas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Potencia del motor:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rueda de repuesto</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>221</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ROJO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AZUL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AMARILLO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VERDE</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Extras:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mat_Et">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MostrarVehiculos">
     <attribute name="title">
      <string>Ver Vehiculos</string>
     </attribute>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricula:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de Ruedas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>kit de reparacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Combustible:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rueda de Repuesto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mas bocinas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Burbuja:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Portavasos:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Color">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Potencia</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Pot">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Rep">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Kit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Bu">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Boc">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Por">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tipo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushAnterior">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>460</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Anterior</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushSig">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>460</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Siguiente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushFoto">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>80</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Foto</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
